<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdCalibrantPropertiesDialog</class>
 <widget class="QDialog" name="QxrdCalibrantPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibrant Properties...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>458</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>alpha (deg):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>c (ang):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_C"/>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_A"/>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QCheckBox" name="m_IsUsed">
     <property name="text">
      <string>Used?</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>b (ang):</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>beta (deg):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_B"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>a (ang):</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>gamma (deg):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="m_Name"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Desc:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="m_Description"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Symmetry:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QComboBox" name="m_Symmetry"/>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_Alpha"/>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_Beta"/>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="m_Gamma"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QxrdCalibrantPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QxrdCalibrantPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
